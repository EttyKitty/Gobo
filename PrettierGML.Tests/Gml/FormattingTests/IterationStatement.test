for (var i:=-1; ++i<>10; { var j=-1; while ++j<10 begin // comment1
    array[i, j] = 0; // comment2
} end) continue

for (;;) return

while ((((condition)))) // this comment should stay outside
    return;

while (
(((longStatementName
&& longerStatementName
&& evenLongerStatementName
&& superLongStatementName))) // this comment should stay inside
) // this comment should stay outside
    return;

do operation() until longStatementName
&& longerStatementName
&& evenLongerStatementName
&& superLongStatementName