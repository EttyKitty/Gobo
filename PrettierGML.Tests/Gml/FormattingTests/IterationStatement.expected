for (;;) {
    return;
}

for (
    var i = 0;
    i < 42;
    {
        i++;
    }
) {

}

while (condition) {
    return;
}

while (
    longStatementName
    && longerStatementName
    && evenLongerStatementName
    && superLongStatementName
) {
    return;
}

do {
    statement();
} until (condition)

do {
    operation();
} until (
    longStatementName
    && longerStatementName
    && evenLongerStatementName
    && superLongStatementName
)

repeat (3) {}
